.gitignore

```diff
@@ -80,13 +80,13 @@ lib-cov
 # Coverage directory used by tools like istanbul
 coverage
 
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+# Grunt intermediate storage (https://gruntjs.com/creating-plugins#storing-task-files)
```

Comment:
Shall I open an issue if I was able to return the nginx version in the home or shall I contact you privately ? (Yeah I know it's off topic, I didn't know where to ask this)